benchmarks/io/checkpoint_simulator/CMakeLists.txt

```diff
@@ -6,11 +6,6 @@ ecbuild_add_executable (
   SOURCES checkpoint_simulator.F90
   DEFINITIONS USE_MPI)
 
-target_link_libraries (${exe} PRIVATE MAPL.shared MPI::MPI_Fortran FARGPARSE::fargparse ESMF::ESMF )
+target_link_libraries (${exe} PRIVATE MAPL.shared MPI::MPI_Fortran FARGPARSE::fargparse ESMF::ESMF OpenMP::OpenMP_Fortran)
```

Comment:
Not to change now, but I'm wondering if we should start putting one argument per line to keep the CMake lines readable?   Just a style thing.

Reply:
Maybe. I know that gersemi:

https://github.com/BlankSpruce/gersemi

is a thing. Let me fiddle with it and see.